apps/server/src/modules/tool/tool-launch/service/launch-strategy/lti11-tool-launch.strategy.spec.ts

```diff
@@ -281,10 +366,18 @@ describe('Lti11ToolLaunchStrategy', () => {
 
 				expect(result).toEqual(
 					expect.arrayContaining([
+						new PropertyData({
+							name: 'roles',
+							value: `${LtiRole.INSTRUCTOR},${LtiRole.LEARNER}`,
+							location: PropertyLocation.BODY,
+						}),
 						new PropertyData({ name: 'lis_person_name_full', value: userDisplayName, location: PropertyLocation.BODY }),
 						new PropertyData({ name: 'user_id', value: userId, location: PropertyLocation.BODY }),
 					])
 				);
+				expect(result).not.toEqual(
+					expect.arrayContaining([expect.objectContaining({ name: 'lis_person_contact_email_primary' })])
```

Comment:
I like how you are testing, that something is not there as this is the whole reason we are doing this 👍